applications/bci_visualization/operators/stream.py

```
@@ -0,0 +1,50 @@
+        self._channels: ChannelInfo
+
+    def setup(self, spec: OperatorSpec) -> None:
+        spec.output("samples")
+
+    def start(self) -> None:
+        self._stream.start()
+        self._channels = self._stream.get_channels()
+        self._iter = self._stream.stream_nirs()
+
+    def compute(
+        self, op_input: InputContext, op_output: OutputContext, context: ExecutionContext
```

Ruff (0.14.10)

Unused method argument: `op_input`

(ARG002)